api/ingests_api/src/test/scala/uk/ac/wellcome/platform/storage/ingests/api/IngestsApiFeatureTest.scala

```diff
@@ -90,6 +92,7 @@ class IngestsApiFeatureTest
                      |    }
                      |  },
                      |  "createdDate": "${ingest.createdDate}",
+                     |  "lastModifiedDate": "${ingest.lastModifiedDate.get}",
```

Comment:
We're 100% sure there's something in the `Option`?

Reply:
Yep – because `lastModifiedDate` is populated if and only if the `events` field is non-empty, which it is.

Also, this is test code – if this doesn’t contain a `lastModifiedDate`, that’s something interesting we’d like to know about!